src/pages/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { NextPage } from "next";
+import { useAppBridge } from "@saleor/app-sdk/app-bridge";
+import { Button } from "@saleor/macaw-ui";
+import dynamic from "next/dynamic";
+import { useEffect, useState } from "react";
+
+const ClientContent = dynamic(() => import("../DashboardActions"), {
+  ssr: false,
+});
+
+/**
+ * This is page publicly accessible from your app.
+ * You should probably remove it.
+ */
+const IndexPage: NextPage = () => {
+  const { appBridgeState } = useAppBridge();
+  const [mounted, setMounted] = useState(false)
+
+  useEffect(() => {setMounted(true)},[])
+
+  return (
+    <div>
+      <h1>Welcome to Saleor App Template (Next.js) 🚀</h1>
+      <p>
+        This is a <strong>minimal</strong> boilerplate you can start with, to create an app
```

Comment:
```suggestion
        This is a boilerplate you can start with, to create an app
```

Bold statement. Pretty sure I can create app in one file :D Nevertheless, this version is a big improvement!